.env.local.example

```diff
@@ -0,0 +1,4 @@
+# Currently required by Token Vault E2E test automation
+VITE_AM_CLIENT_ID=WebOAuthClient
+VITE_AM_URL=https://localhost:9443/am
+VITE_AM_REALM=root
```

Comment:
Currently just adding a separate `.env` for our Token Vault E2E, so we don't overcomplicate the `.env` we use for all our sample apps.